### PR TITLE
docs(qa): Update QA coding challange

### PR DESCRIPTION
**Pull request by PrinceSoni83**
Add end to end scenario for automation from tradeling.com
Remove the old scenario which is not suitable for Cypress.

### CODE REVIEW

**Review by shrmaky — DISMISSED** [review]
LGTM 🚀

File: qa-automation/readme.md [hunk 1]
```diff
@@ -19,29 +19,37 @@ Your task is to write some end-to-end tests with [Cypress](https://www.cypress.i
...
-   1. Make sure the Total field has the correct value (Total = price \* Amount)
-   1. Verifies that you can't submit the form since you are not logged in
-   
-1. Write a data last end-to-end (WebSockets) test that
-   1. Starts on the [Pair trading view](https://www.binance.com/en/trade/ETH_BTC)
-   1. Under the hood, the UI uses WebSockets to fetch tickers
-   1. Verifies that the data is loaded in the tickers,
-   1. Verifies there is a consistent stream of data.
-   1. Verifies the time it takes the socket connection to connect is less than 1 second
+   1. Starts on the [Tradeling home page](https://tradeling.com/)
```
Comment (devypt) [inline qa-automation/readme.md]: make them use the dev link or any other public website, they can't use our products to write automation test.

File: qa-automation/readme.md [hunk 2]
```diff
@@ -19,29 +19,37 @@ Your task is to write some end-to-end tests with [Cypress](https://www.cypress.i
...
+   1. On the nav bar click the category link Food & Beverage
+   1. Click on Meat & Poultry
+   1. Capture the total number of results from breadcrumbs
+   1. Verify that the total results equals the sum of product count on each page
 
+1. Write a second end-to-end test that
+   1. Starts on the [Tradeling home page](https://tradeling.com/)
```
Comment (devypt) [inline qa-automation/readme.md]: same everywhere

**Review by devypt — CHANGES_REQUESTED** [review]

File: qa-automation/readme.md [hunk 3]
```diff
@@ -19,29 +19,37 @@ Your task is to write some end-to-end tests with [Cypress](https://www.cypress.i
...
+   1. On the nav bar click on the category link Office & Stationery
+   1. Click on Office Supplies & Stationery
+   1. Select Writing Instrument on filter by product type
+   1. Select Ballpoint Pens on filter by product
+   1. Apply filter by price between 50 and 60
+   1. Verify that the total results are correct as per the filter applied
+
+1. Write a third end-to-end test that
+   1. Starts on the [Tradeling home page](https://mytrade.link/en)
```
Comment (shrmaky) [inline qa-automation/readme.md]: apologies, i missed this, pls rectify this link

File: qa-automation/readme.md (hunk 1 above)
Comment (shrmaky) [reply qa-automation/readme.md]: sure thanks for the feedback @devypt

**Review by shrmaky — APPROVED** [review]

File: qa-automation/readme.md (hunk 1 above)
Comment (PrinceSoni83) [reply qa-automation/readme.md]: Binance website link included

**Review by devypt — APPROVED** [review]

**Pull request merged**